src/nimble.nim

```diff
@@ -233,10 +233,16 @@ proc buildFromDir(
     raise newException(NimbleError,
         "Nothing to build. Did you specify a module to build using the" &
         " `bin` key in your .nimble file?")
-  var args = args
-  let nimblePkgVersion = "-d:NimblePkgVersion=" & pkgInfo.version
-  for path in paths: args.add("--path:\"" & path & "\" ")
-  var binariesBuilt = 0
+  var
+    binariesBuilt = 0
+    args = args
+  args.add "-d:NimblePkgVersion=" & pkgInfo.version
+  for path in paths:
+    args.add("--path:" & path.quoteShell)
```

Comment:
Nit: making these minor syntax changes just makes the git history harder to follow and IMO it was fine the way it was.